client/Components/GameBoard/GameBoard.scss

```diff
@@ -97,6 +97,7 @@
 
     &.our-side {
         margin: 0;
+        z-index: 20;
```

Comment:
we have variables in _variables.scss for this :)

Reply:
Ah, okay - nice. I've made a change that hopefully makes sense now then.